src/modules/umount/README.md

```diff
@@ -0,0 +1,17 @@
+### Umount Module
+---------
+This module represents the last part of the installation, the unmounting of partitions used for the install.  It is also the last place where it is possible to copy files to the target system, thus the best place to copy an installation log.
+
+To be able to use the copy an installation log of this module you will need to include a log creation to your launcher script or add it to the used calamares.desktop, example of a launcher script:
+
+```
+#!/bin/sh
+sudo /usr/bin/calamares -d > installation.log 
+```
+Example desktop line:
+```
+Exec=sudo /usr/bin/calamares -d > installation.log
```

Comment:
One more note about this, Calamares is not run as user, so the log ends up in the root dir, much harder for users to see.
But no issue to adjust this commit to make 
```/root/.cache/Calamares/Calamares/Calamares.log``` the default and add the option for a sudo calamares -d log as another option